pkgs/development/python-modules/unicode-slugify/default.nix

```diff
@@ -18,15 +18,22 @@ buildPythonPackage rec {
     sha256 = "25f424258317e4cb41093e2953374b3af1f23097297664731cdb3ae46f6bd6c3";
   };
 
+  patches = [
+    ./use_pytest_instead_of_nose.patch
+    (fetchpatch {
+      url = "https://github.com/mozilla/unicode-slugify/pull/41/commits/a18826f440d0b74e536f5e32ebdcf30e720f20d8.patch";
+      hash = "sha256-B27psp0XI5GhoR0l5lFpUOh88hHzjJYzJS5PnIkfFws=";
+    })
```

Comment:
This is a merged PR from 2013; probably not what you want? (Also, we should use `/commit/` rather than `/pull/<number>/commits/`; it’s stabler.)

Reply:
Er, sorry, I messed up the URL. Forget the part about the 2013 PR.

Reply:
Oh right so
`https://github.com/mozilla/unicode-slugify/commit/a18826f440d0b74e536f5e32ebdcf30e720f20d8`
is the same as
`https://github.com/mozilla/unicode-slugify/pull/41/commits/a18826f440d0b74e536f5e32ebdcf30e720f20d8`